src/ucs/sys/sys.c

```diff
@@ -406,7 +407,7 @@ size_t ucs_get_phys_mem_size()
     static size_t phys_pages = 0;
 
     if (phys_pages == 0) {
-        phys_pages = ucs_sysconf(_SC_PHYS_PAGES);
+        phys_pages = ucs_max(ucs_sysconf(_SC_PHYS_PAGES), 1);
```

Comment:
need to check for -1 explicitly, and return SIZE_MAX

Reply:
okay